### DISCUSSION TITLE
Unable to Upload Media Assets

### DISCUSSION THREAD

**Discussion by JeffHochberg** [requarks/wiki#6095] [Help / Questions] [answered]
Hello,

I deployed Wiki.js using Docker Compose as described here: https://docs.requarks.io/install/docker

I have some content I would like to add to posts on my site - I found the Media Assets section in the online documentation: https://docs.requarks.io/guide/assets

Per the documentation:

```
All editors have a button to open the Assets dialog.
```

I'm primarily using the Markdown editor, but I've also checked the Visual Editor - neither provides a mechanism for uploading assets as is described here:

https://docs.requarks.io/editors

Either I'm blind, or the media uploader option is simply not present.

Is the assets option simply not available in the Docker flavor of MWiki.js?

**Comment by NGPixel** [comment, marked as answer]
<img width="497" alt="image" src="https://user-images.githubusercontent.com/15522395/214986131-f7d79c9f-36b7-4bea-8d11-0a97832dccfb.png">

<img width="963" alt="image" src="https://user-images.githubusercontent.com/15522395/214986324-63893053-4047-4500-bc9d-0782d5d93cea.png">
